CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ We use _breaking :warning:_ to mark changes that are not backward compatible (re
 ### Changed
 
 - [#3496](https://github.com/thanos-io/thanos/pull/3496) s3: Respect SignatureV2 flag for all credential providers.
+- [#3539](https://github.com/thanos-io/thanos/pull/3539) Query:  API calls to Query component honour per-request timeout (URL parameter)
```

Comment:
nit: add period to the end of the sentence

Reply:
Thanks! This suggested change is taken care.